src/hotspot/share/prims/methodComparator.cpp

```diff
@@ -75,8 +75,8 @@ bool MethodComparator::args_same(Bytecodes::Code const c_old,  Bytecodes::Code c
   case Bytecodes::_multianewarray : // fall through
   case Bytecodes::_checkcast      : // fall through
   case Bytecodes::_instanceof     : {
-    u2 cpi_old = s_old->get_index_u2();
-    u2 cpi_new = s_new->get_index_u2();
+    int cpi_old = s_old->get_index_u2();
+    int cpi_new = s_new->get_index_u2();
```

Comment:
I had to change these two lines because BytecodeStream::get_index_u2 returns an int, so got the warning and this didn't need to be declared with u2.  get_index_u2() could be fixed to return a u2 but I didn't want to go that far as no casts were involved in this change.

Reply:
I think this change looks "wrong" - the indices are supposed to be u2's, if the function returns an int that seems an error.